src/guide/1/getting-started/first-steps/snippets/create.js

```diff
@@ -0,0 +1,25 @@
+// load the Kuzzle SDK module
+const {
+  Kuzzle,
+  WebSocket
+} = require('kuzzle-sdk');
+/* instantiate a Kuzzle client, this will automatically
```

Comment:
No, you need to connect manually: `await kuzzle.connect();`

This comment isn't useful enough, I propose that you delete it.